celeryapp.py

```diff
@@ -0,0 +1,41 @@
+from celery import Celery
```

Comment:
This file was used in conjunction with docker-compose.yml to test amqp and redis backends on some manual tests.